packages/core/package.json

```diff
@@ -1,4 +1,7 @@
 {
+  "browser": {
+    "./dist/commonjs/shims/node/assetCryptography": "./dist/commonjs/shims/browser/assetCryptography.js"
```

Comment:
Let's organize shims by context instead of placing them all in a `shims` directory.